
For me it loooks like standard behaviour that the linker does not investigate
indirect library dependencies to resolve symbols - maybe other system compiler/
linker are strangely patched ... So we specify the dependencies the code
exposes to let the linker succeed linking the final object ...

  - Rene Rebe <rene@exactcode.de>

--- ./patches/src680/system-mozilla-xmlsecurity-libs.diff.orig	2005-04-18 14:06:01.000000000 +0200
+++ ./patches/src680/system-mozilla-xmlsecurity-libs.diff	2005-04-19 09:40:04.000000000 +0200
@@ -0,0 +1,13 @@
+--- solenv/inc/libs.mk.vanilla	2005-04-19 09:17:36.000000000 +0200
++++ solenv/inc/libs.mk	2005-04-19 09:19:26.000000000 +0200
+@@ -155,7 +155,9 @@
+ NSS3LIB=-lnss3
+ NSPR4LIB=-lnspr4
+ PLC4LIB=-lplc4
+-NSSCRYPTOLIBS=$(LIBXML2LIB) $(XMLSECLIB) $(XMLSECLIB-NSS) $(NSS3LIB) $(NSPR4LIB) $(PLC4LIB)
++PLDS4LIB=-lplds4
++SOFTOKN3LIB=-lsoftokn3
++NSSCRYPTOLIBS=$(LIBXML2LIB) $(XMLSECLIB) $(XMLSECLIB-NSS) $(NSS3LIB) $(NSPR4LIB) $(PLC4LIB) $(PLDS4LIB) $(SOFTOKN3LIB)
+ AUTOMATIONLIB=-lsts$(OFFICEUPD)$(DLLPOSTFIX)
+ SVLLIB=-lsvl$(OFFICEUPD)$(DLLPOSTFIX)
+ SVMEMLIB=
